io/net/Network_config.py

```diff
@@ -110,7 +110,8 @@ def test_networkconfig(self):
                     self.fail("mismatch in mtu")
             if 'MTU' in val:
                 cmd, mtu_val = val.split('MTU:')
-                mtu_val, cmd = mtu_val.split(' ')
+                mtu_val, cmd = mtu_val.split(' M')
```

Comment:
I do not find any difference between SLES and Ubuntu.

### SLES: ###
```
linux-k12h:~ # ifconfig eth0 | grep MTU
          UP BROADCAST RUNNING MULTICAST  MTU:1500  Metric:1

linux-k12h:~ # ip link show eth0
2: eth0: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc pfifo_fast state UNKNOWN mode DEFAULT group default qlen 1000
    link/ether ea:e3:86:8d:2f:02 brd ff:ff:ff:ff:ff:ff
```

### UBUNTU: ###
```
# ifconfig wc0 | grep MTU
          UP POINTOPOINT RUNNING NOARP MULTICAST  MTU:1322  Metric:1

# ip link show wc0
9: wc0: <POINTOPOINT,MULTICAST,NOARP,UP,LOWER_UP> mtu 1322 qdisc pfifo_fast state UNKNOWN mode DEFAULT group default qlen 500
    link/none 
```